### PR TITLE
TG-130: Timesheet

### PR DESCRIPTION
**Pull request by JuanBSantomero**
1.  Validations with Joi on add and edit for Time-Sheet.
2.  Absolute import for routes.

### CODE REVIEW

File: src/Components/TimeSheets/Add/index.jsx [hunk 1]
```diff
@@ -1,135 +1,165 @@
...
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm({
+    mode: 'onBlur',
+    resolver: joiResolver(schema)
+  });
+
+  console.log(errors);
```
Comment (martinalbrnz) [inline src/Components/TimeSheets/Add/index.jsx:34]: ojo el clog

File: src/Components/TimeSheets/Add/index.jsx [hunk 2]
```diff
@@ -1,135 +1,165 @@
...
+            <label htmlFor="description"> Description </label>
             <input
+              {...register('description', { required: true })}
               type="text"
               placeholder="Description"
-              value={description}
-              onChange={(e) => setDescription(e.target.value)}
             />
+            {errors.description?.type === 'string.empty' && <p>{errors.description.message}</p>}
+            {errors.description?.type === 'string.min' && <p>{errors.description.message}</p>}
```
Comment (martinalbrnz) [inline src/Components/TimeSheets/Add/index.jsx:83]: Missing max() validation message

**Comment by Clarosluciano** [comment]
# Test Result: 
Validations: fail
Functionalities: pass

# Bugs found: 
1. The validations in these two inputs are not triggered, date and hours loaded can be placed in an exaggerated way:
image
**Expected behavior:** Validations should appear.

2. The modal title should be better arranged:
image
**Expected behavior:**  "timesheet" should not be on different lines

**Comment by mpueblas** [comment]
Juan fiajte que vercel no te está aprobando la rama

**Review by GinaSchP — APPROVED** [review]

**Review by Clarosluciano — COMMENTED** [review]
# Test result: 
Functionalities: fail

# Bugs found:
1. In table, tasks don't appears when update a timesheet. First you have to reload the page:
image
image
After reloading:
image
**Expected behavior:** The task should appear when the timesheet is finished editing without the need to reload the page.

## PD:
This error appears in the console:
image

**Review by Clarosluciano — APPROVED** [review]

**Review by mpueblas — APPROVED** [review]

**Pull request merged**